<ui version="4.0" >
 <class>HistogramTab</class>
 <widget class="QWidget" name="HistogramTab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>565</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="ButtonGroup1_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Histogram Properties</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>11</number>
      </property>
      <property name="topMargin" >
       <number>11</number>
      </property>
      <property name="rightMargin" >
       <number>11</number>
      </property>
      <property name="bottomMargin" >
       <number>11</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <widget class="QLabel" name="TextLabel5_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Num bins:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>_numBins</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QLabel" name="TextLabel4_5_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>To:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>_max</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="TextLabel1_10" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Data vector:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="4" >
       <widget class="Kst::VectorSelector" native="1" name="_vector" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="TextLabel1_4_2_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>From:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>_min</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3" >
       <widget class="QLineEdit" name="_min" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>The lower limit of the lowest bin.  Values below this limit will not be counted in any bin.</string>
        </property>
        <property name="text" >
         <string>-10.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QLineEdit" name="_max" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>The upper limit of the highest bin.  Values above this limit will not be counted in any bin.</string>
        </property>
        <property name="text" >
         <string>10.0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QSpinBox" name="_numBins" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>The number of bins in the Histogram.</string>
        </property>
        <property name="maximum" >
         <number>1000000</number>
        </property>
        <property name="singleStep" >
         <number>5</number>
        </property>
        <property name="value" >
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="3" >
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="AutoBin" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis" >
           <string>Set the upper and lower limits, and the number of bins to a reasonable default.</string>
          </property>
          <property name="text" >
           <string>A&amp;uto Bin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="_realTimeAutoBin" >
          <property name="text" >
           <string>&amp;Real-time auto bin</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="ButtonGroup5" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Y Axis Normalization</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>11</number>
      </property>
      <property name="topMargin" >
       <number>11</number>
      </property>
      <property name="rightMargin" >
       <number>11</number>
      </property>
      <property name="bottomMargin" >
       <number>11</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="_normIsPercent" >
        <property name="whatsThis" >
         <string>The Y axis of the histogram reports the percent of the samples in the bin.</string>
        </property>
        <property name="text" >
         <string>&amp;Percent in bin</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="_normIsFraction" >
        <property name="whatsThis" >
         <string>The Y axis of the histogram reports the fraction of samples in the bin.</string>
        </property>
        <property name="text" >
         <string>&amp;Fraction in bin</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="_normIsNumber" >
        <property name="whatsThis" >
         <string>The Y axis of the histogram reports the number of samples in the bin.</string>
        </property>
        <property name="text" >
         <string>&amp;Number in bin</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QRadioButton" name="_normPeakIs1" >
        <property name="whatsThis" >
         <string>The Y axis of the histogram is normalized so that the highest bin is 1.</string>
        </property>
        <property name="text" >
         <string>Peak bin = &amp;1.0</string>
        </property>
        <property name="shortcut" >
         <string>Alt+1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="Kst::CurveAppearance" native="1" name="_curveAppearance" />
   </item>
   <item row="3" column="0" >
    <widget class="Kst::CurvePlacement" native="1" name="_curvePlacement" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Kst::VectorSelector</class>
   <extends>QWidget</extends>
   <header>vectorselector.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::CurvePlacement</class>
   <extends>QWidget</extends>
   <header>curveplacement.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::CurveAppearance</class>
   <extends>QWidget</extends>
   <header>curveappearance.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_vector</tabstop>
  <tabstop>_min</tabstop>
  <tabstop>_max</tabstop>
  <tabstop>_numBins</tabstop>
  <tabstop>AutoBin</tabstop>
  <tabstop>_realTimeAutoBin</tabstop>
  <tabstop>_normIsNumber</tabstop>
  <tabstop>_curveAppearance</tabstop>
  <tabstop>_curvePlacement</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
